packages/node_modules/@webex/internal-plugin-encryption/test/integration/spec/kms.js

```diff
@@ -10,6 +10,8 @@ import WebexCore from '@webex/webex-core';
 import testUsers from '@webex/test-helper-test-users';
 import uuid from 'uuid';
 
+const debug = require('debug')('kms');
```

Comment:
Do we have some sort of standard or documentation on these different debug levels?

Reply:
There isn't really. This particular way is used sparingly within the SDK. I find it useful because the DEBUG logs are colored differently so they contrast against the verbose logs. 

Once the log epic is addressed, these may not be needed.